app/components/list-item/index.test.tsx

```diff
@@ -0,0 +1,23 @@
+import { render, screen } from '@testing-library/react'
+import ListItem from '.'
+import { ListItemType } from '@/app/app-types'
+
+describe('List-item component: test cases', () => {
+  const items: ListItemType[] = [
+    {
+      id: '1',
+      name: 'Laundry',
+    },
+  ]
+  test('render list-item component', () => {
+    render(<ListItem items={items} />)
+    const listItemEl = screen.getByTestId('list-item-wrapper')
+    expect(listItemEl).toBeInTheDocument()
+  })
+
+  test('load the list elements', () => {
+    render(<ListItem items={items} />)
+    const listItemEl = screen.getAllByRole('listitem')
+    expect(listItemEl.length).toBe(1)
```

Comment:
Try for value 2 or 3 as for 1 we have already covered in above test case

Reply:
Better to test if label is rendered with correct value.

Reply:
Added the test case for
-  2 test list items
- label is rendered with right value or not